src/workerd/api/tests/streams-test.js

```diff
@@ -500,6 +471,292 @@ export const finalReadOnInternalStreamReturnsBuffer = {
   },
 };
 
+// Test that canceling a stream rejects body consume function
```

Comment:
Non-blocking observation: Same comment as in streams-respond-test and streams-r2-patterns-test, it looks like there's many more test cases leftover internally.